.travis.yml

```diff
@@ -1,18 +1,32 @@
 language: php
 
+sudo: false
+
 php:
     - 7.0
     - 7.1
+    - 7.2
     - nightly
 
 env:
+    - SYMFONY_VERSION=2.7.* # LTS
+    - SYMFONY_VERSION=2.8.* # LTS
     - SYMFONY_VERSION=3.2.*
     - SYMFONY_VERSION=3.3.*
+    - DEPENDENCIES='dev' SYMFONY_VERSION='3.4.*@dev' # LTS
     - SYMFONY_VERSION=dev-master
 
-before_script:
+matrix:
+    exclude:
+        - php: 7.0
+        - env: SYMFONY_VERSION=dev-master # requires PHP ^7.1.3
```

Comment:
👍  nice!

Reply:
strange... it didn't exclude with `php: 7.0` but excluded with `php: 7` :confused: now it's ok... have a change to have 100% green builds :slightly_smiling_face: